bin/kalite

```diff
@@ -58,6 +58,10 @@ else:
     if kalite_path and not os.path.exists(os.path.join(kalite_path, 'kalite', '__init__.py')):
         kalite_path = None
 
+    installed_path = os.path.join(sys.prefix, 'share', 'kalite')
```

Comment:
Would it be good for this to retrieve `ROOT_DATA_PATH` here, to avoid duplication of this `path.join`?